contrib/github-authorized-keys-non-docker.service

```diff
@@ -0,0 +1,18 @@
+[Unit]
+Description=GitHub Authorized Keys
+
+Requires=network-online.target
+After=network-online.target
+
+[Service]
+User=root
+TimeoutStartSec=0
+EnvironmentFile=/etc/secrets/%p
```

Comment:
Let's use /etc/defaults since I think that's a more conventional path. Thoughts?